automated_exports.go

```diff
@@ -0,0 +1,45 @@
+package recurly
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"net/http"
+)
+
+// AutomatedExportsService manages the interactions for automated exports.
+type AutomatedExportsService interface {
+	// Get retrieves export file.
+	//
+	// https://dev.recurly.com/docs/download-export-file
+	Get(ctx context.Context, date string, fileName string) (*AutomatedExport, error)
```

Comment:
Let's make `date` `time.Time` and have the function handle the formatting internally so that the library always makes valid API calls.

Reply:
Updated